File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/VirtualOrderVO.kt
package com.sdkwork.backend

data class VirtualOrderVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val orderId: Int? = null,
    val orderSn: String? = null,
    val success: Boolean? = null,
    val message: String? = null
)

File: sdkwork-backend-sdk-go/types/virtual_order_vo.go
package types

// 虚拟商品订单VO
type VirtualOrderVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	OrderId int `json:"orderId"`
	OrderSn string `json:"orderSn"`
	Success bool `json:"success"`
	Message string `json:"message"`
}
